tower-http/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tokio-util = { version = "0.6", optional = true, default_features = false, featu
 tower = { version = "0.4.1", optional = true }
 tracing = { version = "0.1", default_features = false, optional = true }
 percent-encoding = { version = "2.1.0", optional = true }
+headers = { version = "0.3", optional = true }
```

Comment:
I'm not sure we should depend on `headers`. Using the stuff in `http::headers` is fine imo. Not worth adding another dependency for.

Reply:
I'll have a look on how to remove this dependency